tests/test.ts

```diff
@@ -142,6 +143,37 @@ describe.skip('Test clasp deploy function', () => {
   });
 });
 
+describe.skip('Test clasp versions function', () => {
+  it('should list versions correctly', () => {
+    const result = spawnSync(
+      'clasp', ['versions'], { encoding: 'utf8' },
+    );
+    expect(result.stdout).to.contain('~ ');
+    expect(result.stdout).to.contain(' Versions ~');
```

Comment:
This doesn't seem like a very strong test. Can we create a version, then ensure the version we created is the same as the version listed?